pkg/scheduler/api/types.go

```diff
@@ -16,226 +16,8 @@ limitations under the License.
 
 package api
 
-import (
-	"time"
-
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
-
 const (
 	// DefaultPercentageOfNodesToScore defines the percentage of nodes of all nodes
 	// that once found feasible, the scheduler stops looking for more nodes.
 	DefaultPercentageOfNodesToScore = 50
```

Comment:
we can still move this to apis/ but not to staging/...

Reply:
Done.